cristin-biobank/src/main/java/no/unit/nva/biobank/model/nva/BiobankMaterial.java

```diff
@@ -0,0 +1,33 @@
+package no.unit.nva.biobank.model.nva;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import no.unit.nva.biobank.model.cristin.CristinBiobankMaterial;
+
+public class BiobankMaterial {
+    public static final String MATERIAL_CODE = "materialCode";
+    public static final String OTHER_MATERIAL_DESC = "otherMaterialDescription";
```

Comment:
@torarnet I think description would be enough here, or?

Reply:
We should ask @brinxmat about domain model.

Also, here is a class for ExternalSource, found in project module:
https://github.com/BIBSYSDEV/nva-cristin-service/blob/main/cristin-project/src/main/java/no/unit/nva/cristin/projects/model/nva/ExternalSource.java

Reply:
Looks like you can reuse ExternalSource from project module. Move it to commons.

Reply:
Yes, it was you that commented on ExternalSource in BiobankMaterial comment :)

Reply:
I was not able to write a comment in the other comment, had to write a comment here instead. dont know why